main.go

```
@@ -44,8 +44,8 @@ func main() {
 	kingpin.HelpFlag.Short('h')
 	kingpin.Parse()
 
-	log.Println("Starting exporter", version.Info())
-	log.Println("Build context", version.BuildContext())
+	log.Info("Starting exporter ", version.Info())
+	log.Info("Build context ", version.BuildContext())
 
 	config = loadConfig(*configFile)
```

Maybe you should return an error from `loadConfig` and centralize error management in `main` ? :-)

---

Yeah ! done that !